.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Attention à ces fichiers propres aux IDEs et OSs, ils ne devraient pas être commit. Tu peux [te configurer un `.gitignore` global](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) pour ignorer ce genre de fichiers sur tout tes projets.

Tu peux d'ores-et-déjà y inclure sous Mac:

```
.DS_Store
.vagrant
/vendor
```

Reply:
Merci! C'est tout bon 👍🏽

Reply:
@ogizanagi on revert ce fichier du coup ?

Reply:
Je comprends pas trop où il est